core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt

```diff
@@ -1192,6 +1194,7 @@ abstract class CoreReaderFragment :
   }
 
   private fun closeTab(index: Int) {
+    if (currentTtsWebViewIndex == index) onReadAloudStop()
```

Comment:
Please use `if` condition with curly braces to maintain the project's code style.

Reply:
I have made the changes @MohitMaliFtechiz